hapi19/server.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const glue = require('@hapi/glue');
+const path = require('path');
+const pem = require('pem');
+const { navRoutes } = require('@contrast/test-bench-utils');
+
+const { PORT = 3000, HOST = 'localhost', SSL } = process.env;
+const isHttps = SSL === 1;
+
+const manifest = {
+  server: {
+    debug: { request: ['error', 'uncaught'] },
+    port: PORT,
+    host: HOST,
+    routes: {
```

Comment:
this right here is the only difference between the hapi18 and hapi19 apps - they switched to multipart support being off by default - https://github.com/hapijs/hapi/issues/3920

Reply:
this is weird, no seeing why they turned this off but good catch